README.md

```diff
@@ -48,6 +48,7 @@ You will need development headers/libraries for the following:
 * libhttp_parser [libhttp-parser-dev].
 * sqlite3 [libsqlite3-dev]
 * libolm [libolm-dev] (if not available, compile with `make MATRIX_NO_E2E=1`)
+* libgcrypt20-dev
```

Comment:
this list is on the format "<library name> [<package that provides it>]". Please could you stick to the convention?

Reply:
OK~ I will add the brackets and submit a new PR, then.

Reply:
please just push updates to your branch, no need for a new PR

Reply:
https://help.github.com/en/articles/about-pull-requests